bangazonapi/models/store.py

```diff
@@ -1,13 +1,11 @@
 from django.db import models
-from .customer import Customer
+from .customer import User
```

Comment:
If it doesn't break your code, I would change this back to Customer, here and on line 8, so that existing Store-related code does not need to be changed.